capstone-app/src/app/modules/onboarding/content/content.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'app-content',
+  templateUrl: './content.component.html',
+  styleUrls: ['./content.component.scss']
+})
+export class ContentComponent implements OnInit {
+  computerPhotos = ['assets/gmaprGen.gif', 'assets/mapsSearchbar.gif', 'assets/markerCustomBetter.gif',
+    'assets/mapTypeThemes.gif', 'assets/photos.gif', 'assets/language.gif', 'assets/furtherReading.gif'];
+  mobilePhotos = ['assets/mobileGmapr.gif', 'assets/mobileSearch.gif', 'assets/mobileCustomMark.gif',
+  'assets/mobileMapTheme.gif', 'assets/mobilePhotos.gif', 'assets/mobileLanguage.gif', 'assets/mobileFurtherReading.gif'];  
+  displayPhotos: string[] = [];
+  panelObjs = [
+    {
+      title: 'GMapr Description',
+      photos: '',
+      description: 'Learn more about GMapr',
+      paragraph: 'GMapr provides a snapshot in history of cities from over seventy-five years ago.',
+    },
+    {
+      title: 'Maps Search Bar',
+      photos: '',
+      description: 'Learn more about the Maps search bar',
+      paragraph: 'Use the search bar to search up any city in the world.',
+    },
+    {
+      title: 'Map Custom Markers',
+      photos: '',
+      description: 'Learn more about how to make custom markers',
+      paragraph: 'Click any on the map to create custom markers. To remove custom marker either use the remove all custom marker button or right click the custom marker.',
+    },
```

Comment:
Somewhere in the maps, we do need to include that you can click on a marker to view photos of it. It's an important part of our webapp that we shouldn't gloss over.